models/patientdelivery.js

```diff
@@ -0,0 +1,108 @@
+const { Model } = require('sequelize');
+
+module.exports = (sequelize, DataTypes) => {
+  class Patientdelivery extends Model {
+    /**
+     * Helper method for defining associations.
+     * This method is not a part of Sequelize lifecycle.
+     * The `models/index` file will call this method automatically.
+     */
+    static associate(models) {
+      // define association here
+      Patientdelivery.belongsTo(models.Patient);
+      Patientdelivery.hasMany(models.Ambulance);
+      // Patientdelivery.hasMany(models.Hospital);
+    }
+  }
+  Patientdelivery.init(
+    {
+      patientdelivery_uuid: {
+        field: 'patientdelivery',
+        type: DataTypes.UUID,
+        primaryKey: true,
+        autoIncrement: true,
+      },
+      ambulance_uuid: {
```

Comment:
@komerela for the foreign key property names, check out Francis's comment in `models/hospitalAdministrator.js`. Sequelize expects a specific format (e.g. in this case `AmbulanceId`)